src/AzureBlobStorageUpdateSession/UpdateSession/AzureStorageUpdateSessionDiModule.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
```

Comment:
Was this just renamed, hard to review when they don't show side by side, are there any changes to it?

Reply:
Yes, sadly GitHub does not detect that this is a rename. There was some changes in wiring dependencies and they are tested in end to end tests.